tests/integration/test_interaction.py

```diff
@@ -207,12 +207,17 @@ def assert_placed_item(self, item_value, zone_title, assessment_mode=False):
             self.assertDraggable(item_value)
             self.assertEqual(item.get_attribute('class'), 'option')
             self.assertEqual(item.get_attribute('tabindex'), '0')
-            self.assertEqual(item_description.text, 'Placed in: {}'.format(zone_title))
+            description = 'Placed in: {}'
         else:
             self.assertNotDraggable(item_value)
             self.assertEqual(item.get_attribute('class'), 'option fade')
             self.assertIsNone(item.get_attribute('tabindex'))
-            self.assertEqual(item_description.text, 'Correctly placed in: {}'.format(zone_title))
+            description = 'Correctly placed in: {}'
+
+        if isinstance(zone_title, list):
```

Comment:
@arbrandes Under what circumstances would `zone_title` be a `list`? It might make sense to add a comment explaining that above this line.